ote_sdk/ote_sdk/entities/shapes/rectangle.py

```diff
@@ -67,8 +68,14 @@ def __init__(
             modification_date=modification_date,
         )
 
+        is_valid = True
         for (x, y) in [(x1, y1), (x2, y2)]:
-            self._validate_coordinates(x, y)
+            is_valid = is_valid and self._validate_coordinates(x, y)
+        if not is_valid:
+            warnings.warn(
+                f"Invalid {type(self).__name__} : x1={x1}, y1={y1}, x2={x2}, y2={y2}",
```

Comment:
Please suppress warnings in tests OR change warning message to start from `f"{type(self).__name__} coordinates`